src/app/copy/inline-copy/inline-copy-config.ts

```diff
@@ -0,0 +1,6 @@
+import { CopyConfigBase } from '../copy-config-base';
+
+/**
+ * A config containing properties for inline copy
+ */
+export class InlineCopyConfig extends CopyConfigBase {}
```

Comment:
I believe this can be removed

Reply:
I can, is it preferred to reference CopyConfigBase instead of InlineCopyConfig in the tests and demo files? For example

https://github.com/patternfly/patternfly-ng/pull/351/files#diff-b0ba8ab503cb8f15ddb2a2aa71762883R26 and

https://github.com/patternfly/patternfly-ng/pull/351/files#diff-cbab9d748295daf6283ab1fc63c2ead9R21